### PR TITLE
Documentation for yamaha_musiccast Component

### PR DESCRIPTION
**Pull request by jalmeroth**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#9258<home-assistant PR number goes here>

### CODE REVIEW

File: source/_components/media_player.yamaha_multicast.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: "Yamaha MusicCast Receivers"
+description: "Instructions how to integrate Yamaha MusicCast Receivers into Home Assistant."
```
Comment (frenck) [inline source/_components/media_player.yamaha_multicast.markdown]: **Spelling & Grammar**:
- `Instructions how to`-> `Instructions on how to`

File: source/_components/media_player.yamaha_multicast.markdown [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+```
+Configuration variables:
+
+- **name** (*Optional*): Name of the device
+- **host** (*Required*): IP address or hostname of the device
+- **port** (*Optional*): UDP Port
+
+A few notes:
+
+- Currently, this component supports powering on/off, mute, volume control and source selection. Playback controls, for instance play and stop are available for sources that supports it.
```
Comment (frenck) [inline source/_components/media_player.yamaha_multicast.markdown]: **Spelling & Grammar**:
- Missing comma after `control` -> `volume control, and source selection`
- Missing comma after `instance` -> `for instance, play and stop`
- Singular subject: `supports` -> `support`

**Comment by jalmeroth** [comment]
@frenck thanks for your feedback! Corrected it. 👍

**Comment by frenck** [comment]
Thanks for fixing those issue so quickly @jalmeroth! 🥇

**Comment by frenck** [comment]
Parent PR is merged, merging this PR as well.
Thanks, @jalmeroth!

**Pull request merged**